iso15118/secc/main.py

```diff
@@ -1,21 +1,39 @@
 import asyncio
 import logging
+import sys
+import yaml
 
 from iso15118.secc import SECCHandler
 from iso15118.secc.controller.simulator import SimEVSEController
 from iso15118.shared.exificient_exi_codec import ExificientEXICodec
 
 logger = logging.getLogger(__name__)
+args = sys.argv
 
 
 async def main():
     """
     Entrypoint function that starts the ISO 15118 code running on
     the SECC (Supply Equipment Communication Controller)
     """
-    sim_evse_controller = await SimEVSEController.create()
+
+    global args
+
+    default_config_file = "conf_default.yaml"
+
+    try:
+        with open(args[1]) as f:
+            config_param = yaml.load(f, Loader=yaml.FullLoader)
+        print("Loading configuration parameters from " + args[1])
+        logger.info(f"Loaded configuration parameters from: {args[1]}")
+    except:
```

Comment:
what is the specific exception that is raised? I would assume "FileNotFoundError"?

Reply:
Yes. Additionally, if no configfile is passed as argument, line 25 will result in an IndexError

> with open(args[1]) as f: